secure/key-manager/src/counters.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use libra_secure_push_metrics::{define_counters, Counter};
+use once_cell::sync::Lazy;
+use std::sync::Arc;
+
+// Metrics for the key manager. Uses the "libra_key_manager" prefix for all counters.
+define_counters![
+    "libra_key_manager",
+    (
+        full_consensus_rotations: Counter,
+        "counts the number of full consensus key rotations performed by the key manager"
+    ),
+    (
+        rotation_transaction_resubmissions: Counter,
+        "counts the number of times the key manager had to resubmit a rotation transaction to the blockchain"
+    ),
+    (
+        no_actions_required: Counter,
```

Comment:
same

Reply:
This will be per all keys (i.e., the execution model is that the key manager will wake-up, check all key types and immediately sleep if no action is required). Individual actions per key type will be specific metrics (.e.g, completed_consensus_key_rotations, or completed_network_key_rotations etc.).  Does this make sense?